python/ql/test/library-tests/frameworks/django-orm/dummy_import.py

```diff
@@ -0,0 +1,5 @@
+# to force extractor to see files. since we use `--max-import-depth=1`, we use this
+# "fake" import that doesn't actually work, but tricks the python extractor to look at
+# all the files
```

Comment:
Is this something either we or customers need to be aware of when analysing projects in the wild?

Reply:
no. we run the extractor in a different way when used in tests and with the codeql cli -- this is sort of annoying, but the way it is :neutral_face: :shrug: (I looked into changing this some while back, but it was not easy)

Reply:
Cool, thanks.